### DISCUSSION TITLE
Can't open 'config/enoceanmqtt.devices':  I AM GOING CRAZY....

### DISCUSSION THREAD

**Discussion by einfachtimo** [mak-gitdev/HA_enoceanmqtt#128] [General]
Hello and sorry fot the beginner problem. I just have the problem, that i can't open 'config/enoceanmqtt.devices' and i have NO IDEA what i am doing wrong....

<img width="502" alt="Bildschirmfoto 2024-02-17 um 22 11 59" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/cefe4ea0-b415-46d7-a746-92a7650dad0b">
<img width="1022" alt="Bildschirmfoto 2024-02-17 um 22 11 53" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/8cf73e4a-bf0f-4e8e-977a-caf5c44df3df">
<img width="730" alt="Bildschirmfoto 2024-02-17 um 22 11 43" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/84e5b1e7-7014-4e69-af74-12c4182e0c62">

Hope you could help me...

**Comment by mak-gitdev** [comment]
Hi @einfachtimo, 

In the addon configuration, you have an entry called `device_file` which is currently set to `/config/enoceanmqtt.devices`.
You should set it to `/config/config/enoceanmqtt.devices` instead.

The explanation is that the path to your device file is `/homeassistant/config/enoceanmqtt.devices`.
In the addon, the `/homeassistant` folder is seen as being a folder called `/config`. So from the addon's point of view, your device file is located at `/config/config/enoceanmqtt.devices`.

**Comment by einfachtimo** [comment]
Thank you SOOO MUCH! That was the trick!

**Comment by einfachtimo** [comment]
Can i aski you something else? I am a really fresh beginner here and doing my first steps. I installed the official eltako integration and you addon. And i think its working now:

<img width="1020" alt="Bildschirmfoto 2024-02-18 um 10 48 52" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/4554ba14-69d9-47a4-8f50-470809e2bb10">

Is there a simple way to check if its sending signals? Because i am looking for new devices in the PCT14 Tool, i isn´t seeing any new virtual buttons... 
<img width="363" alt="Bildschirmfoto 2024-02-18 um 10 51 51" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/e91cd5da-a1a0-461c-8db6-0e74e2d87956">
<img width="809" alt="Bildschirmfoto 2024-02-18 um 10 51 35" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/20aad6d9-4b3d-4a2e-b14d-c9c05d7c3a25">
<img width="789" alt="Bildschirmfoto 2024-02-18 um 10 51 33" src="https://github.com/mak-gitdev/HA_enoceanmqtt/assets/388141/10b30e22-227f-4111-91e4-747ca2d18b97">

Thank you !!!

**Reply by mak-gitdev** [reply]
@einfachtimo, 

Everything looks ok so simply generate some traffic from your virtual rockers.
Two possibilities:
- Use the select entity `action`; the telegram is sent as soon as you select one action.
- Set your rocker configuration using switches `AI`, `AO`, `BI`, `BO` and then press `send`.

![crop_1708407020099.jpg](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/73297795/0378c7d3-8ec4-432a-8312-11e685d4f584)
